apps/backend/src/db/migrations/20250805171700_super_admin_seed.ts

```diff
@@ -0,0 +1,47 @@
+import { type Knex } from "knex";
+
+const TableName = {
+	GROUPS: "groups",
+	USERS: "users",
+} as const;
+
+const GROUP_KEY = "admins";
+
+const ColumnName = {
+	EMAIL: "email",
+	ID: "id",
+	KEY: "key",
+} as const;
+
+type IdRecord = { id: number };
+
+const SUPERADMIN_EMAIL = "superadmin@mail.com";
+
+async function down(knex: Knex): Promise<void> {
+	await knex(TableName.USERS).where(ColumnName.EMAIL, SUPERADMIN_EMAIL).del();
+}
+
+async function up(knex: Knex): Promise<void> {
+	await knex.transaction(async (trx) => {
+		const group = await trx<IdRecord>(TableName.GROUPS)
+			.select(ColumnName.ID)
+			.where(ColumnName.KEY, GROUP_KEY)
+			.first();
+
+		if (!group || !group.id) {
```

Comment:
if it's found it will have id
```suggestion
		if (!group) {
```